package.json

```diff
@@ -28,7 +28,7 @@
     "test": "mocha-chrome test/index.html",
     "test-types": "tsc --project ./jsconfig.json",
     "dist": "cp -r src/* dist/ && npm run-script uglify && gzip -9 -k -f dist/htmx.min.js > dist/htmx.min.js.gz && exit",
-    "ws-tests": "cd ./test/ws-sse && node ./server.js",
+    "ws-tests": "cd ./test/ws-sse && node ./server.mjs",
```

Comment:
This was failing. I realized afterwards it wasn't necessary for testing (was running off master initially, which was quietly skipping the sse tests), kept it anyway.